### ISSUE TITLE
Support for Deno runtime

### ISSUE THREAD

**Issue by arafatamim** [grantholle/moviedb-promise#47]
Is support for Deno planned? I see that Axios is a major dependency of this project, and Deno doesn't really have that, so compatibility might be a hard thing to achieve.

**Comment by grantholle** [comment]
Yeah Deno support could be added relatively easily. I got really pumped for Deno and my interest has sort of waned. I'll look into it!

**Comment by hendursaga** [comment]
Any update on this?

**Comment by grantholle** [comment]
Yes, I'm in the process of adding this package to the [JSR](https://jsr.io/) registry, but haven't finished yet. Once that's finished, you can use this package in Node.js, Deno, Bun, Cloudflare Workers, and more.
